### PR TITLE
[native] Fix for ssh-keyscan

### PR DESCRIPTION
**Pull request by Mionsz**
Fix for a bug that I have created in `scripts/release-centos-dockerfile/Dockerfile` with connection to `ssh-keyscan`.

This is a fix to: https://github.com/prestodb/presto/issues/19074

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/scripts/release-centos-dockerfile/Dockerfile
Comment (kgpai) [reply presto-native-execution/scripts/release-centos-dockerfile/Dockerfile:59]: cc: @Mionsz

**Comment by v-jizhang** [comment]
@prestobot kick off tests

**Review by bikramSingh91 — APPROVED** [review]

**Comment by kgpai** [comment]
Validated that the docker image builds now.

File: presto-native-execution/scripts/release-centos-dockerfile/Dockerfile
Comment (Mionsz) [reply presto-native-execution/scripts/release-centos-dockerfile/Dockerfile]: yes - sorry for not responding, to much other tasks currently.

**Comment by Mionsz** [comment]
@kgpai thanks for the fix - I had no time to even check my PRs

**Review by majetideepak — APPROVED** [review]

**Review by gggrace14 — APPROVED** [review]

**Comment by gggrace14** [comment]
Hi @Mionsz are you able to merge this PR now? Is the Rebase and merge button enabled?

**Comment by Mionsz** [comment]
@gggrace14 No - someone should do it for me as I possibly do not have enough rights. Fell free.

**Comment by mshang816** [comment]
> Hi @Mionsz are you able to merge this PR now? Is the Rebase and merge button enabled?

@gggrace14 This PR touches things outside of presto-native so it needs an approval form committer to be merged. I will try to get someone to look at it.

**Comment by mshang816** [comment]
> @gggrace14 No - someone should do it for me as I possibly do not have enough rights. Fell free.

@Mionsz there is a header check error from the checks, can you take a look to see if it can be fixed.

**Comment by kgpai** [comment]
I believe this pr isnt getting closed because this file : [.github/workflows/recurring-native.yml](https://github.com/prestodb/presto/pull/19090/files#diff-f269ddf7d78c31669ceb8cc0b4b888b4e394dddf0b887472d2b843b496a99da5) is present. Its empty and I believe not required.

**Comment by kgpai** [comment]
@mshang816 This should get picked up by the train I believe now.

**Pull request merged**
